Assets/Main/Create New/MenuManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+
+public class MenuManager : MonoBehaviour {
+
+	public GameObject toolsMenu; 
+	public GameObject newObjectMenu; //Initialized in unity
+	public GameObject editSessionMenu; //Initialized in unity
+
+	private MainDirector director;
+	private bool menuLoaded;
+
+	void Start() {
+		director = gameObject.GetComponent<MainDirector>();
```

Comment:
Relating the to issue stated in other file, a check for not null is most likely unnecessary.

Reply:
it not